systemtest/src/test/java/io/strimzi/systemtest/operators/FeatureGatesIsolatedST.java

```diff
@@ -77,6 +79,8 @@ public class FeatureGatesIsolatedST extends AbstractST {
     @Tag(INTERNAL_CLIENTS_USED)
     public void testKRaftMode(ExtensionContext extensionContext) {
         assumeFalse(Environment.isOlmInstall() || Environment.isHelmInstall());
+        assumeTrue(TestKafkaVersion.compareDottedVersions("3.5.0", Environment.ST_KAFKA_VERSION) != 1);
```

Comment:
Maybe you can add the comment as well so that we find it based on the issue number?

Reply:
Yeah, good catch